feature_annotations.yaml

```diff
@@ -0,0 +1,27 @@
+# This code-annotations configuration file supports OEP-17, Feature Toggles.
+
+source_path: ./
+report_path: reports
+safelist_path: .annotation_safe_list.yml
+coverage_target: 50.0
+annotations:
+  documented_elsewhere:
+    - ".. documented_elsewhere_name:":
+  feature_toggle:
+    - ".. toggle_name:":
+    - ".. toggle_type:":
+        choices: [switch, rollout, group, ConfigurationModel, feature_flag]
```

Comment:
What are the semantics of the `toggle_type` field?  It currently seems to be a mix of `use_cases` plus technical implementation.  If it's for technical implementation, then should the choices be the following?
```
waffle_switch, waffle_flag, configuration_model, django_setting
```

Reply:
@pdpinch replied to that on the old PR: https://github.com/edx/edx-platform/pull/19715#discussion_r273288758 .  I think we can definitely iterate on this to make it better, but that might be something we can leave to the Tools team.

Reply:
Thanks @albemarle.  I may be wrong, but I don't believe the Tools team will work on updating the Application code itself.  They will work on the automation of parsing and creating reports.

I see that Peter's comment refers to @robrap's [comment and suggestion](https://github.com/edx/edx-platform/pull/19715/files#r252737849), which works for me as well:

```
toggle_implementation choices: [ConfigurationModel, WaffleFlag, WaffleSwitch, DjangoSetting] 
```

It's unclear what value `toggle_type` will provide,  given that `toggle_implementation` and `toggle_use_cases` may be sufficient.

Reply:
@albemarle  We should also include `CourseWaffleFlag` as an implementation choice.